springfox-swagger-common/src/main/java/springfox/documentation/swagger/csrf/CsrfTokenWebMvcLoader.java

```diff
@@ -0,0 +1,112 @@
+/*
+ *
+ *  Copyright 2015 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package springfox.documentation.swagger.csrf;
+
+import org.springframework.util.StringUtils;
+import org.springframework.web.cors.CorsUtils;
+import org.springframework.web.util.WebUtils;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * The csrf token loader for web-mvc
+ *
+ * @author liuxy
+ */
+public class CsrfTokenWebMvcLoader implements CsrfTokenLoader<MirrorCsrfToken> {
```

Comment:
Move to `springfox-spring-webmvc`. This doesnt belong on `springfox-swagger-common`